test/test_bowtie2.py

```diff
@@ -293,8 +293,11 @@ def testRemoveDuplicates(self, existsMock):
         bt.removeDuplicates()
         log = fp.getvalue()
         self.assertTrue(log.endswith("\nRemoving marked duplicates.\n"))
+        # Note the two spaces in the following. This is due to the way the "-b"
+        # argument is inserted when BAM is being made (which is not the case
+        # here).
         self.assertEqual(
-            "$ samtools view -b -F 1024 '/tmp/xxx/result.sam' "
+            "$ samtools view  -F 1024 '/tmp/xxx/result.sam' "
```

Comment:
Do you have two spaces between `view` and `-F`?

Reply:
Yes. There is a comment in the code before that to explain why (but I guess you don't see it in the GitHub output):

```python
        # Note the two spaces in the following. This is due to the way the "-b"
        # argument is inserted when BAM is being made (which is not the case
        # here).
        self.assertEqual(
            "$ samtools view  -F 1024 '/tmp/xxx/result.sam' "
            "> '/tmp/xxx/non-duplicates.sam'",
            e.log[-2],
        )
```